init-tools.cmd

```diff
@@ -75,7 +75,7 @@ if NOT exist "%DOTNET_LOCAL_PATH%" (
 if exist "%BUILD_TOOLS_PATH%" goto :afterbuildtoolsrestore
 echo Restoring BuildTools version %BUILDTOOLS_VERSION%...
 echo Running: "%DOTNET_CMD%" restore "%INIT_TOOLS_RESTORE_PROJECT%" --no-cache --packages "%PACKAGES_DIR%" --source "%BUILDTOOLS_SOURCE%" /p:BuildToolsPackageVersion=%BUILDTOOLS_VERSION% /p:ToolsDir=%TOOLRUNTIME_DIR% >> "%INIT_TOOLS_LOG%"
```

Comment:
Can you fix up this line to match the line below?

Reply:
This still does not build because I have a space in my path.


When I run -
C:\Users\Andrew Kittredge\Source\Repos\machinelearning\src\Native> .\build.cmd.

I get 

CMake Error: The source directory "C:/Users/Andrew Kittredge/Source/Repos/machinelearning/bin/obj/x64.Debug/Native/Kittredge/Source/Repos/machinelearning/src/Native"".

Reply:
@andrewkittredge - I don't understand your comment. Are you saying your change didn't fix the issue you said it is fixing?

My comment here is regarding that this line `echo command` is supposed to be a duplicate of `command`. So if you change the `command` line - we should keep the `echo command` line up-to-date.

Reply:
Was this fixed before merge? @TomFinley , @eerhardt